discovery-provider/src/models/models.py

```diff
@@ -582,6 +582,20 @@ def __repr__(self):
 count={self.count})>"
 
 
+class AggregateMonthlyPlays(Base):
+    __tablename__ = "aggregate_monthly_plays"
```

Comment:
can you comment what this table will be used for and that timestamp is date_trunc month?